python/grass/jupyter/display.py

```diff
@@ -0,0 +1,101 @@
+
+# MODULE:    grass.jupyter.display
+#
+# AUTHOR(S): Caitlin Haedrich <caitlin DOT haedrich AT gmail>
+#
+# PURPOSE:   This module contains functions for non-interactive display
+#            in Jupyter Notebooks
+#
+# COPYRIGHT: (C) 2021 Caitlin Haedrich, and by the GRASS Development Team
+#
+#           This program is free software under the GNU General Public
+#           License (>=v2). Read the file COPYING that comes with GRASS
+#           for details.
+
+import os
+import folium
+import grass.script as gs
+
+class grassRenderer:
+    """
+    The grassRenderer class creates and displays GRASS maps in Jupyter
+    Notebooks. 
+    """
+
+    def __init__(self, width=600, height=400, text_size=12):
```

Comment:
Add env parameter and when set, use it instead of os.environ and also pass it to the other commands.

Reply:
I'm not 100% sure what this would look like. Is this what you're thinking?

`def __init__(self, env=None, ... ):`

What would go in the function then? I imagine and if-else statement?

Reply:
Exactly. If not env then os.environ. Look to grass.script for more examples, but here, it's just that.

Reply:
In my previous comment, I was thinking about storing (the reference to) it as `self._env`. However, to make this work in parallel (so that two of these things can run in parallel) we need to store a copy. See:

https://github.com/ncsu-geoforall-lab/gis714-2021-tangible-landscape/blob/main/website/render_activities.py#L49

Reply:
Ok, I took a swing at this in my latest commit. Let me know if that's in line with what you were thinking.